### PR TITLE
updated venom goblin text

### PR DESCRIPTION
**Pull request by Jensen1989**

### CODE REVIEW

File: pack/sm_encounter.json [hunk 1]
```diff
@@ -1355,12 +1375,14 @@
...
 		"quantity": 1,
 		"scheme_hazard": 1,
+		"scheme_text": "Hinder 2[per_player]. \n <b> When Revealed: </b> Move the glider token to the main scheme with the most threat. Resolve thats schemes <b>Special</b> ability.",
 		"set_code": "venom_goblin",
 		"set_position": 17,
 		"type_code": "side_scheme"
 	},
 	{
 		"boost": 0,
+		"boost_text": "Resolve te","scheme_text": "Resolve the <b>Special</b> ability of the scheme with the glider counter.",
```
Comment (Kamalisk) [inline pack/sm_encounter.json:1385]: I assume the boost text is supposed to be the same as the scheme text.

Comment (Kamalisk) [inline pack/sm_encounter.json:1385]: I assume the boost text is supposed to be the same as the scheme text.

Comment (Jensen1989) [reply pack/sm_encounter.json]: whoops good catch, I will fix

Comment (Jensen1989) [reply pack/sm_encounter.json]: whoops good catch, I will fix

**Comment by Kamalisk** [comment]
thanks for all the fixes.

**Pull request merged**